terraform/variables.tf

```diff
@@ -27,6 +27,12 @@ variable "prefix_name" {
   }
 }
 
+variable "enable_webhook_gclb" {
```

Comment:
Why wouldn't we always just use the gclb? I'm generally opposed to conditionals that change the flow of Terraform because it leads to spaghetti code that becomes impossible to reason about.

Reply:
I cant target a tagged cloud run revision URL from the GCLB, so for CI, I am disabling it.

I didnt want to create a whole separate module just for CI.

Reply:
To elaborate, I am trying to allow multiple PRs to create revisions for a cloud run services, this requires tagging the revision and hitting it by that specific URL.

Reply:
We could just create it and not use it and let the config pass cloud run ingress and let it be changed to all and just the run url directly. Im open to ideas

Reply:
Interesting - we can't route to a specific revision based on a header or query param?

Let's just document it very very clearly inline.

Reply:
Let me research that first, if not ill document.

Reply:
So this COULD be done with a serverless neg URL mask. We arent setup for this and would take a bit of refactoring. I will create an issue and document the current design inline.

Reply:
> I didnt want to create a whole separate module just for CI.

If cloud run is all you need, this is definitely worth trying. But to call out - because revision is only a cloud run thing, you still can't avoid the problem if you need other isolated resources for parallel CI runs. What we've been doing in JVS/pmap is to decompose the e2e setup module into 1) more static resources 2) resources needed isolation. That way, in CI we can still reuse the 2nd module.

Reply:
Sounds good, I created https://github.com/abcxyz/github-metrics-aggregator/issues/45 to track this